react/features/filmstrip/functions.web.js

```diff
@@ -174,6 +177,33 @@ export function calculateThumbnailSizeForVerticalView(clientWidth: number = 0) {
     };
 }
 
+/**
+ * Calculates the size for thumbnails when in vertical view layout
+ * and the filmstrip is resizable.
+ *
+ * @param {number} clientWidth - The height of the app window.
+ * @param {number} filmstripWidth - The width of the filmstrip.
+ * @returns {{local: {height, width}, remote: {height, width}}}
+ */
+export function calculateThumbnailSizeForResizableVerticalView(clientWidth: number = 0, filmstripWidth: number) {
```

Comment:
I don't think there are too many differences between this function and `calculateThumbnailSizeForVerticalView`. Especially if we use the same ratios from interface_config.js instead of `DEFAULT_FILMSTRIP_WIDTH`.

Does it make sense to combine them in one function?

Reply:
Explained [here](https://github.com/jitsi/jitsi-meet/pull/10884#discussion_r800402258) why we don't use the interface config ratios